GeoZarr Client-Side Rendering Narrative (#4)
EOPF-Explorer/narratives: cloud-native-eo.md

# A Journey into Cloud-Native Earth Observation <!--{ as="img" mode="hero" src="https://images.unsplash.com/photo-1633421878925-ac220d8f6e4f?q=80&w=1760&auto=format&fit=crop&ixlib=rb-4.1.0&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D" }-->

## Visualizing GeoZarr
Imagine having instant access to terabytes of satellite imagery without ever needing to download a single file. This is the promise of cloud-native geospatial data.

> 

Reviewer: I'd omit this, for brevity. Cloud-native is explained elsewhere.

```suggestion
```

> In this story, we'll explore the EOPF-Explorer client side rendered GeoZarr. We'll start by connecting to a catalog of Sentinel-2 imagery, find a specific scene that catches our interest, and then—using the magic of GeoZarr and [EOxElements]() based on the latest [OpenLayers release]() we can visualize an item in the browser using web components, and [ipyeoxelements]() in a Python environment. We'll stream that high-resolution data directly onto an interactive map.

Reviewer: A little rewording below - and this seems to be set up to have hyperlinks but they're not there yet?

```suggestion
In this story, we'll explore client-side rendered GeoZarr in Jupyter Notebooks and in the browser, using the same underlying technology. 

We'll start by connecting to a catalog of Sentinel-2 imagery, find a specific scene that catches our interest, and then—using GeoZarr and [EOxElements]() based on the latest [OpenLayers release]() we can visualize an item in the browser using web components, and [ipyeoxelements]() in a Python environment. We'll stream that high-resolution data directly onto an interactive map.
```

## Gathering Our Tools

> Before we embark, we need to pack our digital knapsack. We'll need `httpx` to communicate with the outside world (the API) and `ipyeoxelements`, our lens for viewing the geospatial data.

Reviewer: I'd really like the language to be less try-hard and more to the point.

Let's please drop things like:

> Before we embark, we need to pack our digital knapsack.

Who gets impressed by something like that? Reading that, I feel like the author thinks I'm stupid.

```suggestion
We'll need `httpx` to communicate with the outside world (the API) and `ipyeoxelements`, our lens for viewing the geospatial data.
```

```python
import httpx
import json
from ipyeoxelements import EOxMap, EOxLayercontrol
from ipywidgets import HBox, Layout
```

> ## Charting the Course

Reviewer: ```suggestion
## Selecting the data source
```

> Every expedition needs a destination. Here, we define the coordinates of our data source, the EOPF-Explorer STAC API. We're specifically setting our sights on the `sentinel-2-l2a` collection.

Reviewer: ```suggestion
Here, we define our data source, the EOPF-Explorer STAC API, specifically the `sentinel-2-l2a` collection.
```
